.env.example

```
@@ -491,14 +491,20 @@ LINEAR_GAME_FEEDBACK_LABEL_ID=
 NFT_CONTRACT_ADDRESS=
 # Chain ID where the NFT contract is deployed (defaults to 1 = Ethereum mainnet)
 NFT_CHAIN_ID=1
+# Envio hosted GraphQL endpoint for on-chain NFT ownership checks (required for on-chain gating / secondary transfers).
+# Example: https://indexer.dev.hyperindex.xyz/<project>/v1/graphql
+NFT_INDEXER_GRAPHQL_URL=
 # Enable NFT-gated app access (default: false). When enabled, non-holders are redirected to /nft
 # and most authenticated API routes return 403 unless the user has minted via the Top 100 flow.
 NFT_GATING_ENABLED=false
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NFT_GATING_ENABLED key should go before the NFT_INDEXER_GRAPHQL_URL key

(UnorderedKey)